packages/app-utils/src/components/ConnectionBootstrap.tsx

```diff
@@ -93,7 +93,7 @@ export function ConnectionBootstrap({
         // We send an update with the fetch right away
         onUpdate({
           fetch: () => objectFetcher(descriptor),
-          error: null,
+          status: 'ready',
```

Comment:
The statuses should probably be consts (or an enum, but just a JS object "enum")

Reply:
Any reason to use constants for that instead of just defined in the type in TS?

Reply:
I guess just the convention we've normally used, but a TS string literal union works too.

Reply:
We use string literal unions in lots of places, like ButtonTypes, not sure we wouldn't use them here?
We use constants when it's defined from the JS API, but this status is purely client side.